### DISCUSSION TITLE
feat(Modal/Dialog): Add option to not close  previous modal if user open next one

### DISCUSSION THREAD

**Discussion by anna777x** [dotnetcore/BootstrapBlazor#3606] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

How it works now:
1. user clicks on button which open first modal dialog 
2. user do some action inside first modal dialog
3. next system shows new 'error' modal to user and hides first modal at same time
4. user clicks 'OK' on error modal and after that first modal is showed again

so I need first modal is showed all time until user don't close it by himself

### Describe the solution you'd like

I suggest to make it as parameter -> to hide previous dialogs/modals or not.  
because now looks like new modal close first dialog and it can to fear user.

**Comment by bb-auto** [comment]
@anna777x Thank you for contacting us. We will give feedback later.

**Comment by ArgoZhang** [comment]
@anna777x  thank you for contacting us.

I completely understand your needs and we have discussed this issue very early on. There are some unclear issues regarding interaction.

1. How to handle masks when there are multiple pop-up windows
2. How to close the pop-up window when clicking on the mask
3. Added conflicts between multiple pop-up windows and some other functions.

Can you submit a PR? I can invite you to the project and try to make changes on another branch. If the function is normal, we will merge it into the main branch.

**Comment by anna777x** [comment]
Hi, I'd like to do it, but I couldn't create PR

![image](https://github.com/dotnetcore/BootstrapBlazor/assets/61416594/cb599fd5-6acf-4f2f-b9b8-51e4ec1daf0d)

I've also wanted to push some small fixes, but it doesn't allow to do it as well
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/61416594/1672be99-e16b-4a40-a716-60f208b3a820)

**Comment by ArgoZhang** [comment]
@anna777x I have sent an invite to you. please confirm that. thank you very much.

**Comment by anna777x** [comment]
yes, I'm in project. so what's next?

**Comment by ArgoZhang** [comment]
> yes, I'm in project. so what's next?

create a new branch and push it into the repo.

**Comment by anna777x** [comment]
I've created 'dev-multiple-modals' branch. I think maybe we can start by using Named Cascading Parameter and at leats give chance to close specific modal. But I'll think about it tomorrow. Have a good day!
